templates/bundles/DemosPlanCoreBundle/DemosPlanProcedure/administration_edit.html.twig

```diff
@@ -898,6 +911,31 @@
                                             )
                                             }}
                                         {% endif %}
+
+                                        <dp-input
```

Comment:
Curious question: Both fields are not wrapped within a permission... this means that they're gonna be visible for all projects, along with the rather opinionated microcopy ("Übersichtskarte/Bild", "Laden Sie ein Bild des Bebauungsplanausschnitts auf einer Karte hoch."). Did you take that decision on purpose?